src/sentry/static/sentry/app/components/selectInput.jsx

```diff
@@ -61,17 +63,8 @@ const SelectInput = React.createClass({
   },
 
   render() {
-    let opts = {
-        ref: 'select',
-        disabled: this.props.disabled,
-        required: this.props.required,
-        multiple: this.props.multiple,
-        placeholder: this.props.placeholder,
-        className: this.props.className,
-        value: this.props.value,
-    };
     return (
-      <select {...opts}>
+      <select ref="select" {...this.props}>
```

Comment:
idk if you can do children={} to bind it though. This was just convenience so we stopped missing props